content/en/users/compute/orchestration/ec3/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Elastic Cloud Compute Clusters
+linkTitle: Cloud Compute Clusters
```

Comment:
```suggestion
linkTitle: Elastic Cloud Compute Cluster
```

I think we should avoid dissolving the EC3 branding

Reply:
This is too long for the nav tree. Here users search for "what" not for the official name of a service. We have plenty of space for that in the actual page.